health-claim/src/components/Login/RegisterForm.js

```diff
@@ -29,42 +31,6 @@ export default class RegisterForm extends Component {
     } else if (this.props.loading) {
       return <h1>Loading</h1>;
     }
-    return (
-      <>
-        <h1 className="Page__Title">Sign up</h1>
-        <Box>
-          <form onSubmit={this.handleSubmit} onChange={this.handleChange}>
-            <label className="Input__Label" htmlFor="username">
-              Your username
-            </label>
-            <input
-              className="Input__Text"
-              placeholder="Username"
-              type="text"
-              name="username"
-              id="username"
-            />
-            <br />
-            <label className="Input__Label" htmlFor="password">
-              Password
-            </label>
-            <input
-              placeholder="Password"
-              className="Input__Text"
-              type="password"
-              name="password"
-              id="passowrd"
-            />
-            <hr />
-
-            <input
-              type="submit"
-              value="Sign up"
-              className="Button Button__Green"
-            />
-          </form>
-        </Box>
-      </>
-    );
+    return <RegistrationFormBox />;
```

Comment:
one last thing, do we need to extract this? If we're not passing anything it seems a bit unnecessary